.github/workflows/code-quality.yaml

```diff
@@ -35,16 +35,27 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                package: ${{ fromJson(needs.php-packages-matrix.outputs.packages) }}
+                php-version: [ '8.1', '8.2', '8.3', '8.4']
```

Comment:
`package: ${{ fromJson(needs.php-packages-matrix.outputs.packages) }}` is a leftover from #2764